src/Http/Controllers/LogViewerController.php

```diff
@@ -148,14 +148,10 @@ public function search(Request $request, $date, $level = 'all')
 
         $log     = $this->getLogOrFail($date);
         $levels  = $this->logViewer->levelsNames();
-        $needles = array_filter(explode(' ', $query));
```

Comment:
Is there a reason why removed the `array_filter` and the `unless` ?

Reply:
I replaced it with array_map because I want to change the value of the needle to a lowercase version. From my understanding array_filter will just return the value if the callback returns true.

Reply:
`array_filter` is meant to filter empty string:

Example:

```php
$needles = explode(' ', 'Filter  entries  by  my  query');

dd($needles, array_filter($needles));
```

Reply:
Okay, I see what you are saying. I will wrap the array_map with array_filter. Thanks